src/components/Header.tsx

```diff
@@ -1,44 +1,34 @@
-import NextLogo from './NextLogo'
-import SupabaseLogo from './SupabaseLogo'
+import Link from 'next/link'
+import Image from 'next/image'
 
 export default function Header() {
   return (
-    <div className="flex flex-col items-center gap-16">
-      <div className="flex items-center justify-center gap-8">
-        <a
-          href="https://supabase.com/?utm_source=create-next-app&utm_medium=template&utm_term=nextjs"
-          target="_blank"
-          rel="noreferrer"
-        >
-          <SupabaseLogo />
-        </a>
-        <span className="h-6 rotate-45 border-l" />
-        <a href="https://nextjs.org/" target="_blank" rel="noreferrer">
-          <NextLogo />
-        </a>
+    <header className="w-full bg-base-200 p-4 shadow-md">
+      <div className="container mx-auto flex items-center justify-between">
+        <div className="flex text-xl font-bold">
+          <Image
+            src="/CalendarIcon.png"
+            width="1000"
```

Comment:
this size is really odd here. It intrinsically will be much smaller and the actual file is 150px at best.  Needed?